tsconfig.json

```diff
@@ -16,7 +16,8 @@
     "incremental": true,
     "baseUrl": "./",
     "paths": {
-      "@/*": ["./*"]
+      "@/*": ["./*"],
+      "react": ["./node_modules/@types/react"]
```

Comment:
- why is this required?

Reply:
Please respond to the question first, before marking this as resolved @ashifkhn.